jobbergate-cli/jobbergate_cli/subapps/applications/app.py

```diff
@@ -101,22 +106,28 @@ def list_all(
 @app.command()
 def get_one(
     ctx: typer.Context,
+    id_or_identifier: Optional[str] = typer.Argument(
+        None,
+        help="The specific id number or identifier of the application to be selected.",
+    ),
     id: Optional[int] = typer.Option(
         None,
         "--id",
         "-i",
-        help=f"The specific id of the application. {ID_NOTE}",
+        help=f"Alternative way to specify id. {ID_NOTE}",
```

Comment:
It would be good to trigger a deprecation warning on these options so we can remove them in the future.

Reply:
I agree this would be beneficial. However, I'm concerned that some calls to jobbergate-cli occur within application scripts and other scripts that end-users interact with. This could lead to confusion about the warnings. I'll create a Jira ticket for the Software Dev Team to address these points. Once that's done, we can implement the deprecation warning.

Reply:
That sounds good to me